src/stan/variational/advi.hpp

```diff
@@ -334,6 +520,14 @@ namespace stan {
 
         // draw more samples from posterior and write on subsequent lines
         if (out_stream_) {
```

Comment:
There's a double conditional statement here... I think the `if (out_stream_)` should be removed. @akucukelbir / @dustinvtran agree?